DependencyInjection/Configuration.php

```diff
@@ -35,6 +35,17 @@ public function getConfigTreeBuilder()
         $knownProcessors = $this->knownProcessors;
 
         $rootNode
+            ->beforeNormalization()
+                ->always()
+                ->then(function ($v) {
+                    // Deal with old logger config
+                    if (isset($v['publisher_logger']) && !isset($v['logger'])) {
+                        $v['logger'] = $v['publisher_logger'];
```

Comment:
you should also trigger a deprecation notice here, to let developers know they should update their config